src/notification.c

```diff
@@ -382,6 +359,189 @@ check_notification (GVariant *notification,
   return TRUE;
 }
 
+/* From https://github.com/flatpak/flatpak/blob/master/common/flatpak-run.c */
+G_GNUC_NULL_TERMINATED
+static void
+add_args (GPtrArray *argv_array, ...)
+{
+  va_list args;
+  const char *arg;
+
+  va_start (args, argv_array);
+  while ((arg = va_arg (args, const gchar *)))
+    g_ptr_array_add (argv_array, g_strdup (arg));
+  va_end (args);
+}
+
+static void
+add_env (GPtrArray  *array,
+         const char *envvar)
+{
+  if (g_getenv (envvar) != NULL)
+    add_args (array,
+              "--setenv", envvar, g_getenv (envvar),
+              NULL);
+}
+
+static void
+add_bwrap (GPtrArray *array,
+           const char *input)
+{
+  const char *bwrap;
+
+  bwrap = g_getenv ("FLATPAK_BWRAP");
+  if (bwrap == NULL)
+    bwrap = "/usr/bin/bwrap";
```

Comment:
Not a fan of hardcoding `/usr/bin`, maybe match the configured prefix of flatpak.

Reply:
Alternatively just get the default at configure time, which is probably the most reliable.